packages/abi-typegen/package.json

```
@@ -40,9 +40,10 @@
     "dist"
   ],
   "scripts": {
-    "pretest": "pnpm fuels-forc build -p test/fixtures/forc-projects",
+    "pretest": "pnpm build:forc",
     "build": "tsup",
-    "postbuild": "tsx ../../scripts/postbuild.ts"
+    "postbuild": "tsx ../../scripts/postbuild.ts",
+    "build:forc": "pnpm fuels-forc build -p test/fixtures/forc-projects"
```

Sort of a nit, no pun intended. Sorry. 😓 

```suggestion
    "build:forc": "pnpm fuels-forc build -p test/fixtures/forc-projects"
    "postbuild": "tsx ../../scripts/postbuild.ts",
```

---

I was thinking about this ordering as well when I was writing it and just went with the initial version when I realized that we don't have a standardized way for ordering anything in `package.json`. It would be best to add a package in our `devDependencies` for this and add it as a step in `pnpm lint`.